DSharpPlus/DiscordIntents.cs

```diff
@@ -83,7 +83,7 @@ public enum DiscordIntents
         /// Whether to include guild ban events.
         /// <para>These include <see cref="DiscordClient.GuildBanAdded"/>, and <see cref="DiscordClient.GuildBanRemoved"/>.</para>
         /// </summary>
-        GuildBans = 1 << 2,
+        GuildModeration = 1 << 2,
```

Comment:
why? and why in this PR?

Reply:
It was renamed by Discord but idk why i put it in this PR